docs/glossary.md

```diff
@@ -14,6 +14,8 @@
 ## Battle Terms:
 - **Roll**: How many dice can the unit roll.
 - **Strength**: The number on the dice that determines whether a hit was made. Any number less than or equal to the number is a hit.
+- **Power**: A calculation using the roll and strength that is used for low luck battles.
+- **Attack**: Generally used to reference Strength (such as attackAA, offensiveAttackAA, and attack). But sometimes used to reference Roll (such as maxAAattacks).
```

Comment:
Attack is indeed poorly worded.  Because it can also reference the side (attack vs defense).  I haven't seen it related to power, though.  I've only seen strength, roll, and battle side.

How difficult would it be to rename the xml parameters?  I can open an issue and talk about renaming the properties if this is something that would be acceptable to do.

Reply:
Hmm, I think we should probably strike this then.

Just because we have an XML parameter, does not mean it makes it into the glossary. In theory the glossary should be domain specific, but beyond that the glossary is the definition of what the terms should be, not necessarily what they are (and in some cases the terms are present and poorly used).

Reply:
It would be nice to have some place that stores this information for new developers.  Knowing that "attack" isn't really the correct term and that any code or comments referencing "attack" is out outdated would be useful.

Reply:
A list of slightly incorrect terms might be long, though perhaps useful. I wonder if the effort might be better used to try and unify the terms. For example, once the term is parsed from XML, we control the names of the variables. 

That's my 2 cents, in the meantime we should distinguish between active & current terminology and arguably legacy.